__tests__/app.test.js

```diff
@@ -40,3 +40,44 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  test("200: responds with a single matching review", () => {
+    const REVIEW_ID = 2;
+    return request(app)
+      .get(`/api/reviews/${REVIEW_ID}`)
```

Comment:
In your other tests you've provided an inline value for your endpoint, be consistent, I don't think you gain any extra readability or clarity here by declaring `REVIEW_ID` above.